packages/sdk/src/util/webHost.ts

```diff
@@ -5,12 +5,20 @@
 
 import { resolve as urlResolve } from 'url';
 import * as Restify from 'restify';
+//import './restify.plugins.serveStaticFiles';
```

Comment:
After moving the .d.ts file to the `types` folder and adding it to tsconfig.json, this import statement should work.

Reply:
It doesn't, because that import statement is looking for a `built` version of that file, and it won't exist. I should remove this.

Reply:
If you remove the `./` path portion (and do the other stuff I mentioned), then it should work.